copilot/fsd-form-runner/addons/form-runner-data.yml

```diff
@@ -58,39 +57,64 @@ Resources:
   FormRunnerRedisReplicationGroup:
     Type: AWS::ElastiCache::ReplicationGroup
     Properties:
-      ReplicationGroupDescription: !Sub '${Env} Funding Service Form Runner'
+      ReplicationGroupDescription: !Sub "${Env} Funding Service Form Runner"
       AutomaticFailoverEnabled: true
       AtRestEncryptionEnabled: true
       TransitEncryptionEnabled: true
       AutoMinorVersionUpgrade: true
       MultiAZEnabled: true
       CacheNodeType: cache.t3.micro
-      CacheSubnetGroupName: !Ref 'FormRunnerRedisSubnetGroup'
+      CacheSubnetGroupName: !Ref "FormRunnerRedisSubnetGroup"
       SecurityGroupIds:
-        - !GetAtt 'FormRunnerRedisSecurityGroup.GroupId'
+        - !GetAtt "FormRunnerRedisSecurityGroup.GroupId"
       Engine: redis
       NumCacheClusters: 2
 
   # Redis endpoint stored in SSM so that other services can retrieve the endpoint.
   FormRunnerRedisEndpointAddressParam:
     Type: AWS::SSM::Parameter
     Properties:
-      Name: !Sub '/${App}/${Env}/${Name}/redis'   # Other services can retrieve the endpoint from this path.
+      Name: !Sub "/${App}/${Env}/${Name}/redis" # Other services can retrieve the endpoint from this path.
       Type: String
-      Value: !GetAtt 'FormRunnerRedisReplicationGroup.PrimaryEndPoint.Address'
+      Value: !GetAtt "FormRunnerRedisReplicationGroup.PrimaryEndPoint.Address"
+
+  FormRunnerFormUploadsBucketAccessPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Sid: S3FormUploadBucketActions
+            Effect: Allow
+            Action:
+              - s3:Get*
+              - s3:List*
+              - s3:Describe*
+              - s3:PutObject
+              - s3:PutObjectACL
+              - s3:DeleteObject
```

Comment:
Does the application need to be able to delete files?

Reply:
https://github.com/communitiesuk/funding-service-design-application-store/blob/main/copilot/fsd-application-store/addons/fsd-application-store-cluster.yml 
Yes it's here

Reply:
https://github.com/communitiesuk/digital-form-builder/blob/main/runner/src/server/services/uploadService.ts#L423-L438

As part of [this functionality](https://github.com/communitiesuk/digital-form-builder/blob/main/runner/src/server/services/uploadService.ts#L190-L196)